### DISCUSSION TITLE
Documentation with examples

### DISCUSSION THREAD

**Discussion by mattkeanny** [ip-tools/python-epo-ops-client#83] [Q&A] [answered]
I have no experience in building documentation sites, just wondering what would be a simple way of showing some examples of using the different methods of the library? Perhaps a examples.md file in the docs as a starter?

**Comment by gsong** [comment]
That works. And a link to it from `README`. Whatever work is done in this step can be repurposed later if a decision is made to publish to a doc site.

Although, I'll defer to @amotl since he's guiding the long term vision for this package now.

**Reply by amotl** [reply, marked as answer]
Hi. Thanks Matt. It will certainly be valuable to have a dedicated documentation for this library. We will ramp up a best-practice setup for that.

- GH-139
